py/desitarget/sandbox/cuts.py

```diff
@@ -512,9 +614,13 @@ def apply_sandbox_cuts(objects,FoMthresh=None):
                        primary=primary)
 
     if FoMthresh is not None:
-        elg, FoM = apply_XD_globalerror(objects, FoMthresh, glim=23.8, rlim=23.4, zlim=22.4, gr_ref=0.5,
+        if (FoMthresh!=2.0) :
```

Comment:
What is special about `FoMthresh=2.0`?  It looks like you're trying to distinguish between @jaekor91's XD algorithm and your new RF algorithm, but these should probably be totally separate, right?